MultiSelect.js

```diff
@@ -26,11 +26,13 @@ class MultiSelect extends Component {
     }
   }
 
-  shouldComponentUpdate(nextProps) {
+  shouldComponentUpdate(nextProps, nextState) {
     // This is a performance check as this sometimes gets updated often
-    if (nextProps.selectedOptions.length === this.props.selectedOptions.length) {
+    if (nextProps.selectedOptions === this.props.selectedOptions &&
```

Comment:
Could you make this `if` look like this please:
```javascript
if (
  (variable === variable) &&
  (varible1 === variable1)
) {
  // do things here
}
```
I prefer this more verbose option for multiline stuff